src/components/common/TmSessionExplore.vue

```diff
@@ -88,6 +121,23 @@ export default {
       } else {
         return false
       }
+    },
+    getAddressIcon(addressType) {
+      if (addressType === "explore") return `language`
+      if (addressType === "ledger") return `vpn_key`
+      if (addressType === "extension") return `laptop`
+    },
+    getAddressTypeDescription(addressType) {
+      if (addressType === "explore") return `Explore Mode`
+      if (addressType === "ledger") return `Ledger Nano S`
+      if (addressType === "extension") return `Lunie Browser Extension`
+    },
+    shortenAddress(address) {
```

Comment:
Instead of custom coding I propose you use the filter `formatBech32` from `filters.js` like in `Bech32.vue`

Reply:
Totally agree, `shortenAddress()` replaced by `formatBech32` filter.